apps/climatemappedafrica/src/lib/data/blockify/explore-page.js

```diff
@@ -46,6 +54,10 @@ async function explorePage(block, _api, _context, { hurumap }) {
     id: "explore-page",
     blockType: "explore-page",
     choropleth,
+    hurumapConfig: {
+      hurumapAPIURL,
+      profileId,
+    },
```

Comment:
1. Isn't `profileId` already in the returned `profile`? My understanding is a single app will always use the same profile.
2. As we're "blockify-ing" here, I don't think it makes sense to return `hurumapConfig` again... If we need the `url` at component level, lets return it as such.

Reply:
Yes @kilemensi Some components will need to know about it.

Reply:
Then why aren't we just passing it as just `url` (or `hurumapUrl` if there might be confusion with just `url`). @kelvinkipruto? We're passing everything else as is e.g. `profile`, `rootGeography`, etc.